progs/jubjub.py

```diff
@@ -0,0 +1,315 @@
+from honeybadgermpc.field import GF, GFElement
+from honeybadgermpc.elliptic_curve import Subgroup
+from honeybadgermpc.mpc import Mpc
+
+Field = GF(Subgroup.BLS12_381)
```

Comment:
Umm, I am not usually a fan of global variables. It is very easy for someone to change this from another file which could potentially break a lot of things if more people start to reuse it. I would suggest thinking of another way to use this. Move this within the Jubjub class as a static member?

Reply:
I have noticed that if you expose the variable like this then more people start to reuse it in places which have no connection with Jubjub just because they want to use the field.

Reply:
I agree, It's now a static variable.